ctapipe_io_lst/__init__.py

```diff
@@ -18,16 +18,26 @@
 )
 
 from ctapipe.io import EventSource
-
-from .containers import LSTDataContainer, PixelStatusContainer
-
+from ctapipe.core.traits import Int
+from .containers import LSTDataContainer
+from ctapipe.io.containers import PixelStatusContainer
 
 __all__ = ['LSTEventSource']
 
 
 class LSTEventSource(EventSource):
     """EventSource for LST r0 data."""
 
+    n_gains = Int(
+        2,
+        help='Number of gains at r0/r1 level'
+    ).tag(config=True)
+
+    baseline = Int(
+        250,
```

Comment:
Just by curiosity, why 250?

Reply:
This is actually the value I used for NectarCAM, in the case of the LST EvB3 should be 400 (I just changed it), but actually now (with EvB2) we are not using this value because the baseline subtraction is done at the r0 calibration level